messaging/infinispan-grpc-kafka/src/main/resources/application.properties

```diff
@@ -27,3 +27,5 @@ mp.messaging.incoming.prices.value.deserializer=org.apache.kafka.common.serializ
 # configure apache kafka
 enable.auto.commit=true
 kafka.group.id=my-group
+
+quarkus.log.category."org.infinispan".level=DEBUG
```

Comment:
I'm not against this, but is that really necessary? Any specific reason, please?